uniffi_aries_vcx/core/src/core/unpack_message.rs

```diff
@@ -0,0 +1,34 @@
+use std::sync::Arc;
+
+use aries_vcx::errors::error::{AriesVcxError, AriesVcxErrorKind};
+use vdrtools::domain::crypto::pack::UnpackMessage;
```

Comment:
preferably create your own struct for this rather than importing from libvdrtools. we should try to avoid direct libvdrtools dependencies

Reply:
you could just copy paste libvdrtool's definition for the struct;
```
#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq)]
pub struct UnpackMessage {
    pub message: String,
    pub recipient_verkey: String,
    #[serde(skip_serializing_if = "Option::is_none")]
    pub sender_verkey: Option<String>,
}
```